### PR TITLE
Created the spring boot project with the necessary dependencies

### PR DESCRIPTION
**Pull request by waterflow80**
Created the spring boot project with the necessary dependencies

-  Added the necessary dependencies
-  Added a maven profile 'seqcol' that has credentials needed in src/main/resources/application.properties
    that will be provided at compilation time. (the maven profile is added in pom.xml)

### CODE REVIEW

File: .idea/.gitignore
Comment (sundarvenkata-EBI) [inline .idea/.gitignore:1]: You can take out the .idea directory with [gitignore](https://github.com/EBIvariation/eva-accession/blob/fcb07c98cbcd9c1871a33cbfbb1d0aff9438e780/.gitignore#L12).

**Review by sundarvenkata-EBI — APPROVED** [review]

**Review by apriltuesday — APPROVED** [review]

**Review by nitin-ebi — APPROVED** [review]

**Review by tcezard — APPROVED** [review]

**Comment by tcezard** [comment]
You can merge this PR and rebase the other on top of this one so that they are easier to read.

**Comment by tcezard** [comment]
Before you merge please remove the .idea folder from the PR

**Comment by waterflow80** [comment]
@tcezard 
Should I do the merge ? Because still I don't have write access to this repo.

**Pull request merged**
